src/sagemaker/clarify.py

```diff
@@ -430,6 +693,11 @@ def __init__(
                 Note: For JSON, the JMESPath query must result in a list of labels for each
                 sample.  For JSON Lines, it must result in the label for each line.
                 Only a single label per sample is supported at this time.
+            headers (str): List of column names in the dataset. If not provided, Clarify will
```

Comment:
should it be list of strings (type annotation)? 
```suggestion
            headers ([str]): List of column names in the dataset. If not provided, Clarify will
```

Reply:
Yes, will fix.